File: blog-sever/server-java/src/main/java/com/site/blog/model/entity/SysTimer.java
package com.site.blog.model.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.fasterxml.jackson.annotation.JsonFormat;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

/**
 * @author yanni
 * @date time 2022/6/17 15:47
 * @modified By:
 */
@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class SysTimer {
    @TableId(type = IdType.ASSIGN_ID)
    private  String id;
    private  String timerName;
    private  String actionClass;
    private  String cron;
    private  Boolean status;
    private String remark;
    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss", timezone = "GMT+8")
    private LocalDateTime createTime;
    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss", timezone = "GMT+8")
    private LocalDateTime updateTime;
}

File: blog-sever/server-jpa/src/main/kotlin/com/site/blog/model/entity/SysTimer.kt
package com.site.blog.model.entity

import com.baomidou.mybatisplus.annotation.IdType
import com.baomidou.mybatisplus.annotation.TableId
import com.fasterxml.jackson.annotation.JsonFormat
import java.time.LocalDateTime

/**
 * @author yanni
 * @date time 2022/6/17 15:47
 * @modified By:
 */
data class SysTimer(
    @TableId(type = IdType.ASSIGN_ID)
    var id: String? = null,
    var timerName: String? = null,
    var actionClass: String? = null,
    var cron: String? = null,
    var status: Boolean? = null,
    var remark: String? = null,

    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss", timezone = "GMT+8")
    var createTime: LocalDateTime? = null,

    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss", timezone = "GMT+8")
    var updateTime: LocalDateTime? = null,
)
  
 